src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/catalog.html.twig

```diff
@@ -133,61 +133,45 @@
                 <tr class="column-headers">
                   <th class="small-text" style="width: 8%">
                     {{ "ID"|trans({}, 'Admin.Global') }}
-                    {% include '@Product/CatalogPage/catalog_order_carrets.html.twig' with {
-                      'column': 'id_product'
-                    } %}
+                    {{ ps.carrets('id_product') }}
                   </th>
                   <th class="small-text">
                     {{ "Image"|trans({}, 'Admin.Global') }}
                   </th>
                   <th class="small-text">
                     {{ "Name"|trans({}, 'Admin.Global') }}
-                    {% include '@Product/CatalogPage/catalog_order_carrets.html.twig' with {
-                      'column': 'name'
-                    } %}
+                    {{ ps.carrets('name') }}
                   </th>
                   <th class="small-text" style="width: 9%">
                     {{ "Reference"|trans({}, 'Admin.Global') }}
-                    {% include '@Product/CatalogPage/catalog_order_carrets.html.twig' with {
-                      'column': 'reference'
-                    } %}
+                    {{ ps.carrets('reference') }}
```

Comment:
what is "carret" in this context? what does it mean?

Reply:
hmm its not even english xD, renamed to sortArrows, thanks.